src/exercise/02.html

```diff
@@ -4,22 +4,20 @@
 <body>
   <div id="root"></div>
 
-  <!-- 💣 remove this line
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
-  💣 remove this line -->
 
   <script type="module">
     const rootElement = document.getElementById('root')
 
-    // You're going to re-implement this code using React!
-    // 💣 So go ahead and delete this implementation (or comment it out for now)
-    // These three lines are similar to React.createElement
-    const element = document.createElement('div')
-    element.textContent = 'Hello World' // 💰 in React, you set this with the "children" prop
-    element.className = 'container' // 💰 in React, this is also called the "className" prop
-    // This is similar to ReactDOM.render
-    rootElement.append(element)
+    const helloElement = React.createElement('span', {children: 'Hello'});
+    const worldElement = React.createElement('span', {children: 'World'});
+
+    const elementProps = {className: 'container', children: [helloElement, worldElement]}
```

Comment:
wouldn't this technically output `<span>Hello</span><span>World</span>`? i.e. `HelloWorld` vs the expected:
`Hello World`?

Reply:
@jakewrfoster I was doing the extra credit: https://github.com/alleyinteractive/react-fundamentals/blob/main/src/exercise/02.md
which is looking for ```
<div class="container">
      <span>Hello</span>
      <span>World</span>
</div>```
but I guess I need blank lines after the spans in the markup to provide the space.